Services/Storage/StorageRecords.cs

```diff
@@ -91,11 +92,10 @@ public async Task<StorageRecord> GetAsync(string id)
             }
             catch (DocumentClientException e) when (e.StatusCode == HttpStatusCode.NotFound)
             {
-                this.log.Debug("The resource requested doesn't exist.", () => new { this.storageName, id });
+                this.log.Debug("The resource requested doesn't exist.", () => new{this.storageName, id});
```

Comment:
nit: Isn't 'this.log.Debug("The resource requested doesn't exist.", () => new { this.storageName, id });' the correct formatting?